worker/src/tests/mod.rs

```diff
@@ -49,7 +49,7 @@ pub fn run_enclave_tests(matches: &ArgMatches, port: &str) {
 	if matches.is_present("all") || matches.is_present("ecall") {
 		println!("Running ecall Tests");
 		println!("  testing get_state()");
-		get_state_works(&enclave);
+		get_state_works(&enclave).unwrap();
```

Comment:
Nice catch!

Reply:
This wasn't a bug - I just changed the function to return a result, moving the `unwrap` from inside the test function to the test client 😄

Reply:
Yes but still, I might have forgotten to add it here. :) Although, you probably got a warning of an unused result here.